src/views/SessionView/Whiteboard.vue

```diff
@@ -160,16 +247,40 @@ export default {
     // Set up custom selection handles
     this.setSelectionHandles();
 
+    // disable showing hints on the canvas
+    this.zwibblerCtx.setConfig("showHints", false);
+
     this.zwibblerCtx.on("connected", () => {
       // Set brush tool to default tool
       this.useBrushTool();
 
       // Don't start setting selected tool until connected
       this.zwibblerCtx.on("tool-changed", toolname => {
         this.selectedTool = toolname;
+        this.hideHoveredToolbars();
       });
     });
 
+    this.zwibblerCtx.on("nodes-added", nodes => {
+      if (this.isShapeSelected) this.shapeNodes.push(nodes[0]);
```

Comment:
When using a shape related tool a node is added at the start of the drawing (on mouse down) and at the end of the drawing (on mouse up). We're using this to determine when a user has finished drawing a shape and default to then default to the pick tool

I tried using other event listeners like a `mouseup` and if the selected tool is a shape tool, but the event listener kept getting the tool name for the tool before the current chosen one The `mouseup` event listener might have been firing before the `tool-changed` event listener did so I went with this method here instead

Reply:
Gotcha, that makes sense 👍